src/touchstone/utils/metadata.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Can you move this file to other folder?, something like "examples" in the project's root would be great